### ISSUE TITLE
Buttons in PTabsBar lose focus onClick in Safari

### ISSUE THREAD

**Issue by marcuszierke** [porsche-design-system/porsche-design-system#1722]
Description  
---
Once a button is clicked on the `tabs-bar` in Safari it looks like that the `activeElement` is not the one it's supposed to be. This results in a "positive" bug which means that the user doesn't see the button outline `onClick`.

To Reproduce
---
1. Go to e.g. https://designsystem.porsche.com/v2/components/tabs-bar/examples#active-tab
2. Open your console in Dev Tools
3. Click on any of the buttons in the `tabs-bar`(Tab One, Tab Two or Tab Three)
4. Go to console and type `document.activeElement`

Expected behavior
---
The button should be the `activeElement` like it is when you test it in Chrome
<img width="384" alt="Screenshot 2022-01-17 at 09 53 47" src="https://user-images.githubusercontent.com/23409802/149738057-8c2cd595-cf36-40d0-8c89-ea38432f8b4e.png">

On Safari the `activeElement` is the `body`
<img width="574" alt="Screenshot 2022-01-17 at 09 56 08" src="https://user-images.githubusercontent.com/23409802/149738428-45a1d724-c879-4713-937c-a90e569aff34.png">

Test Environment
---
macOS Monterey (12.1), Safari 15.2
